scripts/newinstall.sh

```diff
@@ -217,6 +214,43 @@ if true; then
 	fi
 fi
 
+##########	Test/warn about the existence of ~/.eups
+
+if true; then
+	if [[ -e ~/.eups ]]; then
+		TIMESTAMP=$(date +%Y%m%d%S)
+		cat <<-EOF
+
+		Problems have been observed when re-installing or upgrading EUPS while
+		preserving a pre-existing ~/.eups directory.  This directory primarily
+		contains cache data and is safe to remove when EUPS is not actively
```

Comment:
Could simply warn that this may be the problem if there are issues later on with the EUPS part of the script and suggest that moving it out the way might help. Rather than "recommending" removal.